utility/src/main/java/org/oppia/android/util/profile/CurrentUserProfileIdIntentDecorator.kt

```diff
@@ -0,0 +1,32 @@
+package org.oppia.android.util.profile
+
+import android.content.Intent
+import org.oppia.android.app.model.ProfileId
+import org.oppia.android.util.extensions.getProtoExtra
+import org.oppia.android.util.extensions.putProtoExtra
+
+private const val PROFILE_ID_INTENT_DECORATOR = "PROFILE_ID_INTENT_DECORATOR"
+
+/** Decorator for wrapping a user's profile [ProfileId] within its intent. */
```

Comment:
```suggestion
/** Decorator that allows an activity to wrap a user's [ProfileId] within its intent. */
```

I wasn't sure what "its" was referring to, so suggesting a rephrasing for clarity.

Reply:
Makes sense updated.